### PR TITLE
READY FOR REVIEW - Fix type issue

### PR DESCRIPTION
**Pull request by posita**

### CODE REVIEW

File: stone/target/tsd_types.py [hunk 1]
```diff
@@ -112,7 +112,8 @@ class TSDTypesGenerator(CodeGenerator):
 
     preserve_aliases = True
 
-    cur_namespace = None
+    # Instance var of the current namespace being generated
+    cur_namespace = None  # type: ignore # TODO: What is the type of this thing?!
```
Comment (braincore) [inline stone/target/tsd_types.py]: It's a `stone.api.ApiNamespace` object.

File: stone/target/js_client.py [hunk 2]
```diff
@@ -44,7 +45,7 @@ class JavascriptClientGenerator(CodeGenerator):
     cmdline_parser = _cmdline_parser
 
     # Instance var of the current namespace being generated
-    cur_namespace = None  # type: ignore # TODO: What is the type of this thing?!
+    cur_namespace = None  # type: ApiNamespace
```
Comment (braincore) [inline stone/target/js_client.py]: Does this need to be marked `Optional[ApiNamespace]` or does mypy infer that from the declaration?

Comment (posita) [inline stone/target/js_client.py]: Probably yes, even though our current mypy tests pass (although I can't quite figure out why this slips by).

Comment (posita) [inline stone/target/js_client.py]: Fixed now.

**Pull request merged**